adyenv6b2ccheckoutaddon/acceleratoraddon/web/webroot/WEB-INF/views/responsive/pages/checkout/multi/selectPaymentMethodPage.jsp

```diff
@@ -17,12 +17,12 @@
 <template:page pageTitle="${pageTitle}" hideHeaderLinks="true">
     <jsp:attribute name="pageScripts">
         <script type="text/javascript" src="${dfUrl}"></script>
-        <script type="text/javascript" src="https://${checkoutShopperHost}/checkoutshopper/sdk/3.10.0/adyen.js"></script>
+        <script type="text/javascript" src="https://${checkoutShopperHost}/checkoutshopper/sdk/3.17.2/adyen.js"></script>
```

Comment:
sepa component changed the field names it returns: https://docs.adyen.com/checkout/release-notes#nov-6th-2020 can you please double check it still works?

Reply:
Indeed, it also needed changes.
Thanks, already pushed a fix.